### DISCUSSION TITLE
ScriptReference/Material-shaderKeywords

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#223] [Notes]
# ScriptReference/Material-shaderKeywords

https://docs.unity3d.com/ScriptReference/Material-shaderKeywords.html

**Comment by mvi** [comment]
This list is returned alphabetically sorted regardless of the keyword order specified by inspecting the material in the debug inspector or by examining the yaml of the material in a text editor. For example you can specify the shader keywords B A C in the debug inspector and this will be visible in the .mat file yaml too, but this property will return A B C
